src/library_pipefs.js

```diff
@@ -92,10 +92,12 @@ addToLibrary({
           currentLength += bucket.offset - bucket.roffset;
         }
 
+#if !MEMORY64
```

Comment:
In Memory64 mode, which is the `buffer` an instance of?

Reply:
I think it an instance of a Proxy object.  See src/runtime_view_proxy.js.

I can be more specific here and add a comment.